app/api/views.py

```diff
@@ -1,34 +1,51 @@
-from flask import flash, jsonify, redirect, render_template
-from ..models import EditableHTML, Api
+from flask import flash, jsonify, redirect, render_template, url_for
+from ..models import EditableHTML, Api, ApiParameterLink, Parameter
 
+from flask_login import login_required
+from sqlalchemy.exc import IntegrityError
+from wtforms.fields import SelectField
+from flask_wtf.file import InputRequired
+
+from .forms import (
+    NewAPIForm,
+    ParameterForm
+)
+
+from .. import db
 from . import api
 
 @api.route('/index')
```

Comment:
Could you change this to '/'? (just to make consistent with the rest of the routes)